internal/pkg/model/manifest.go

```diff
@@ -84,7 +84,7 @@ type ConfigRowManifest struct {
 }
 
 type ConfigManifestWithRows struct {
-	*ConfigManifest
+	ConfigManifest
```

Comment:
Tu bol pointer, a teda ak som do manifestu vlozil prazdny zaznam, ... tak sa hodnota nenastavila a bol tam `nil`, co robil problem na viacerych miestach.

Vyriesene tym, ze to nebude pointer.

Reply:
Potom to uz pracuje spravne, jednodlive `fields` su validovane.